lang/en/docs/installing-dependencies.md

```diff
@@ -5,13 +5,15 @@ layout: guide
 additional_reading_tags: ["dependencies", "package-json", "yarn-lock", "cli-install"]
 ---
 
-If you have just checked out a package from [version control](./version-control), you will need to install those dependencies.
+{% include vars.html %}
```

Comment:
same, this is probably not necessary, or else it should be added to `_layouts/guide`